Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "FillArrays"
 uuid = "1a297f60-69ca-5386-bcde-b61e274b549b"
-version = "0.8.13"
+version = "0.8.14"
```

Comment:
Should be 0.9 since you export new types

Reply:
✓